projects/plugins/boost/app/assets/src/js/features/cornerstone-pages/meta/meta.tsx

```diff
@@ -203,6 +203,162 @@ const PredefinedList: FC< PredefinedListProps > = ( { items } ) => {
+		const pagesToLoad = Math.min( missingDefaults.length, availableSlots );
+		const willTruncate = pagesToLoad < missingDefaults.length;
+
+		return willTruncate
+			? sprintf(
+					/* translators: %1$d is pages that will be loaded, %2$d is total available pages */
+					__( 'Load %1$d of %2$d default pages (plan limit).', 'jetpack-boost' ),
+					pagesToLoad,
+					missingDefaults.length
+			  )
+			: sprintf(
+					/* translators: %d is the number of pages that will be loaded */
+					_n(
+						'Load %d default page from compatible plugins.',
+						'Load %d default pages from compatible plugins.',
+						pagesToLoad,
+						'jetpack-boost'
+					),
+					pagesToLoad
+			  );
+	};
+
+	const loadDefaultValue = () => {
+		// Use pre-calculated values
+		const { missingDefaults, availableSlots } = buttonLogic;
+		const toAppend = missingDefaults.slice( 0, availableSlots );
+		const newPages = [ ...currentPages, ...toAppend ];
+
+		// Update the input value with the combined list
+		onValueChange( newPages.join( '\n' ) );
+
+		// Show appropriate feedback
+		if ( toAppend.length < missingDefaults.length ) {
+			setNotice( {
+				id: 'cornerstone-load-defaults',
+				type: 'error',
+				message: sprintf(
+					/* translators: %1$d is pages loaded, %2$d is total available pages */
+					__( 'Loaded %1$d of %2$d default pages (plan limit reached).', 'jetpack-boost' ),
+					toAppend.length,
+					missingDefaults.length
+				),
+			} );
+		} else {
+			setNotice( {
+				id: 'cornerstone-load-defaults',
+				type: 'success',
+				message: sprintf(
+					/* translators: %d is the number of pages loaded */
+					_n(
+						'Loaded %d default page.',
+						'Loaded %d default pages.',
+						toAppend.length,
+						'jetpack-boost'
+					),
+					toAppend.length
+				),
+			} );
+		}
+
+		recordBoostEvent( 'cornerstone_pages_load_default', {
+			loaded_count: toAppend.length,
+			available_count: missingDefaults.length,
+			was_truncated: toAppend.length < missingDefaults.length ? 'true' : 'false',
+		} );
+	};
+
+	// Simplified button state logic using pre-calculated values
+	const getButtonState = () => {
+		const { hasDefaults, missingDefaults, availableSlots, hasAllDefaults } = buttonLogic;
+
+		if ( ! hasDefaults ) {
+			return {
+				disabled: true,
+				title: __( 'No default pages available. Add pages manually.', 'jetpack-boost' ),
+			};
+		}
+
+		if ( hasAllDefaults ) {
+			return { disabled: true, title: __( 'Default pages are already loaded.', 'jetpack-boost' ) };
+		}
+
+		// Handle case where user has reached plan limit
+		if ( availableSlots === 0 ) {
+			return {
+				disabled: true,
+				title: __( 'Cannot load defaults. Plan limit reached.', 'jetpack-boost' ),
+			};
+		}
+
+		return {
+			disabled: false,
+			title: getTooltipMessage( missingDefaults, availableSlots ),
+		};
+	};
+
+	const buttonState = getButtonState();
+
+	return (
+		<Tooltip text={ buttonState.title }>
```

Comment:
```suggestion
		<Tooltip text={ buttonState.title } delay={ 0 }>
```

While testing, I noticed that the default delay of 700ms wasn't fast enough. I think having it like this would make the UX better as it would immediately show up once the user hovers. Otherwise, they wouldn't know that there's a tooltip until they wait the 700ms (which would probably be accidentally).

Delay 700:

![Image](https://github.com/user-attachments/assets/047a8477-a9f2-4cda-9155-33ba78c31128)

Delay 0:

![Image](https://github.com/user-attachments/assets/bf459686-fb2a-446c-b4a9-e68f37c6370d)

Reply:
Good idea! This is implemented now.